<launch>

  <arg name="bag" />
  <param name="use_sim_time" value="true" />

  <node pkg="rosbag" type="play" name="player" required="true" output="screen" args=" -r 1 --clock $(find datmo)/examples/$(arg bag).bag"/>

  <node type="rviz" name="rviz" pkg="rviz" args="-d $(find datmo)/config/default.rviz" />
  <node pkg="tf2_ros" type="static_transform_publisher" name="link_laser" args="-0.12 0 0 0 0 0 /ego_pose /laser" />

   <node pkg="datmo" type="datmo" name="datmo" output="screen">
      <param name="max_cluster_size" value="50" />
      <param name="lidar_frame" value="laser" />
      <param name="world_frame" value="map" />
      <param name="threshold_distance" value="0.17" />
      <param name="euclidean_distance" value="0.25" />
      <param name="pub_markers" value="true" />
   </node>

</launch>
